db/seeds.rb

```diff
@@ -159,138 +158,135 @@
+      category: categories[17],
+      description: 'На пл. Міцькевича красуються зимові красуні, які зростають досить дико. '\
                  'Ялинкам не надається відповідна форма. Років десять тому тут вже росли ялинки '\
                  'подібного розміру, і замість надання їм форми - їх було викорчувано і насаджено '\
                  'нових, які тепер досягли такого ж росту. Прохання, не викорчовувати їх на цей раз, '\
                  'а надати їм естетичної форми, щоб вони мали однаковий та симетричний вигляд. '\
                  'І надалі слід слідкувати за їх зростанням. Також було б доречним вже до наступний '\
                  'Різдвяних свях їх прикрасити святковою ілюмінацією, адже це одна з основних площ '\
                  'яка засаджена ялинками - одним із символів Різдва.',
-    attachment: Rack::Test::UploadedFile.new(
-        Rails.root.join('spec', 'files', 'xmas_tree.jpg')) },
-  { name: "Забруднені ворота",
-    address: { lat: 48.9211466, lng: 24.7108564,
-               street_address: 'вул. Гетьмана Мазепи 30' },
-    category: categories[20],
-    description: 'На вул. Гетьмана Мазепи, 30, знаходяться занедбані ворота, '\
+      attachment: Rack::Test::UploadedFile.new(
+          Rails.root.join('spec', 'files', 'xmas_tree.jpg')) },
+    { title: "Забруднені ворота",
+      address: { lat: 48.9211466, lng: 24.7108564,
+                 street_address: 'вул. Гетьмана Мазепи 30' },
+      category: categories[20],
+      description: 'На вул. Гетьмана Мазепи, 30, знаходяться занедбані ворота, '\
                  'які псують вигляд вулиці. '\
                  'Прохання очистити ворота. Дякую.',
-    attachment: Rack::Test::UploadedFile.new(
-        Rails.root.join('spec', 'files', 'dirty_2.jpg')) }
+      attachment: Rack::Test::UploadedFile.new(
+          Rails.root.join('spec', 'files', 'dirty_2.jpg')) }
 ]
 
 issues.each do |issue|
-  FactoryGirl.create(:issue, title: issue[:name],
-                             location: issue[:address][:street_address],
-                             latitude: issue[:address][:lat],
-                             longitude: issue[:address][:lng],
-                             address: issue[:address][:street_address],
-                             name: Faker::Name.name_with_middle,
-                             phone: Faker::PhoneNumber.cell_phone,
-                             category_id: categories.index(issue[:category]),
-                             description: issue[:description],
-                             attachment: issue[:attachment],
-                             created_at: Time.now - rand(10).days)
+  FactoryGirl.create(:issue, title: issue[:title],
+                     address: issue[:address],
+                     phone: Faker::PhoneNumber.cell_phone,
+                     category: issue[:category],
+                     description: issue[:description],
+                     attachment: issue[:attachment],
```

Comment:
I didn't. 
We didn't implement full issue modeling in seeds until you've added Geo-location.
Fixed!